core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/messaginglayer/ControlOutputPort.scala

```diff
@@ -17,12 +18,14 @@ import scala.collection.mutable
   * The internal logic can send control messages to other actor without knowing
   * where the actor is and without determining the sequence number.
   */
-class ControlOutputPort(selfID: ActorVirtualIdentity, networkSenderActor: NetworkSenderActorRef) {
+class ControlOutputPort(selfID: ActorVirtualIdentity, networkSenderActor: NetworkSenderActorRef)
+    extends LazyLogging {
   private val idToSequenceNums = new mutable.AnyRefMap[ActorVirtualIdentity, AtomicLong]()
 
   def sendTo(to: ActorVirtualIdentity, event: ControlPayload): Unit = {
     val seqNum = idToSequenceNums.getOrElseUpdate(to, new AtomicLong()).getAndIncrement()
     val msg = WorkflowControlMessage(selfID, seqNum, event)
+    logger.info(s"send $msg to $to")
```

Comment:
in the params, please remove the work event and name it according to our naming style. I think after before merging this PR, do a general search for places where you may have used `event`

Reply:
will do